packages/ui/src/tmp/README.md

```diff
@@ -0,0 +1,5 @@
+# Temporary location
```

Comment:
I added this README as I'm not sure where to put test selectors we need in the extension and the `ui/packages` . More info [here](https://github.com/orgs/leather-wallet/discussions/66#discussioncomment-9595101)